pom.xml

```diff
@@ -185,7 +180,7 @@
                     <plugin>
                         <groupId>org.eclipse.tycho</groupId>
                         <artifactId>tycho-versions-plugin</artifactId>
-                        <version>${tycho.version}</version>
+                        <version>1.4.0</version>
```

Comment:
If no tycho stuff is left, we probably also won't need the tycho-versions-plugin anymire - I guess the "prepare-release" profile can be removed (it was used for the old manual release process).

Reply:
The execution ID `update-version` has been configured to execute the `set-version` goal.
So, I assume that function is used anywhere.

Reply:
I doubt it, but ok, no need to remove it in this PR.